### ISSUE TITLE
[Sign up][Emails] User signup email (with confirmation link) is always sent in default language, not in user's language

### ISSUE THREAD

**Issue by drummer83** [openfoodfoundation/wishlist#469]
## Description

Users signing up through the website may be using a language other than the default language. However we are always sending the email with the confirmation link in the default language only. The user may not be able to read and understand the email.

## Expected Behavior

The email should be sent in the language which the user used during its sign up.

## Actual Behaviour

The email is always sent in the instance's default language.

## Steps to Reproduce

1. Log out.
2. Select a language other than the instance's default language (set in the `.env` or `.env.development` or `.env.staging` file).
3. Sign up a new user account in that language.
4. Check the language of the email you will receive to confirm your account.

## Severity
bug-s5: we can live with it, only a few users impacted

## Your Environment

* Version used: staged openfoodfoundation/openfoodnetwork#11487 
* Browser name and version:
* Operating System and version (desktop or mobile):

**Comment by lin-d-hop** [comment]
@drummer83 why did you move this from the main repo? seems like an s3 bug to me

**Comment by sigmundpetersen** [comment]
> why did you move this from the main repo?

Well, the Epic is also in wishlist, so I guess that's why?

**Comment by RachL** [comment]
Yep the epic is part of old stuff that were move from the main repo to the wishlist. I don't know if you remember @lin-d-hop but the idea was to figure out if there were papercut we culd make out of these old issues.

**Comment by drummer83** [comment]
@lin-d-hop @RachL I created all the issues I found lately in wishlist because I understood that we didn't want contributors from Hackathon jumping on them immediately. If I got that wrong feel free to move them to main repo or let me know.
Also if you think some of them are ready to be picked up, you could move them.

**Comment by RachL** [comment]
Yes you've understood correctly :+1:  And I did move already some of your issues to the main repo based on that :)

In case of bugs we are safe to create them on the main repo straight away. But I guess for example all the tax ones you've discovered can be introduced later, when the current chunck of s2 are done. This would make things easier to read on the main repo in terms of priority. But happy to be wrong.
